charts/thoras/templates/api-server-v2/rbac.yaml

```
@@ -52,7 +52,7 @@ rules:
 - apiGroups:
   - apps
```

This should be a wild card too. Argo Rollouts live in the `argoproj.io` api group
```suggestion
  - '*'
```

---

I updated this to add `argoproj.io` to the list of `apiGroups` and tested it out locally to make sure it worked.